web/viewer.js

```diff
@@ -1972,21 +1972,32 @@ window.addEventListener('pagechange', function pagechange(evt) {
 }, true);
 
 function handleMouseWheel(evt) {
-  var MOUSE_WHEEL_DELTA_FACTOR = 40;
-  var ticks = (evt.type === 'DOMMouseScroll') ? -evt.detail :
-              evt.wheelDelta / MOUSE_WHEEL_DELTA_FACTOR;
-  var direction = (ticks < 0) ? 'zoomOut' : 'zoomIn';
+  var WHEEL_DELTA_FACTOR = 40;
+
+  var delta = evt.deltaY;
+  if (delta !== undefined) {
+    switch (evt.deltaMode) {
+      // case 0: pixels - no change
```

Comment:
Won't this cause the `case 0` to be handled in `default` below, and was that really indented here?